README.md

```diff
@@ -1,6 +1,11 @@
-# knausj_talon
+# community
```

Comment:
```suggestion
# Talon community grammar
```

Reply:
I'm kind of used to the GitHub convention of the first line being the name of the repository, with a description both underneath and in the repository description field. I don't feel strongly but I thought this was the convention unless you had a strong reason not to

Reply:
This seems to be the general convention still, I think it's fine as is.